Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -2180,6 +2182,33 @@ int CLuaElementDefs::SetElementCollidableWith ( lua_State* luaVM )
 }
 
 
+int CLuaElementDefs::SetElementCollidableWithType ( lua_State* luaVM ) {
+    // bool SetElementCollidableWithType ( element theElement, string withType, bool enabled, bool onlyWithCreated = false )  
+    CClientEntity* pEntity = NULL;
+    SString strType;
+    bool bCanCollide = true;
+    bool bOnlyWithCreated = false;
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadUserData ( pEntity );
+    argStream.ReadString ( strType );
+    argStream.ReadBool ( bCanCollide );
+    argStream.ReadBool ( bOnlyWithCreated, false );
+
+    // Verify the arguments
```

Comment:
You might have copy-pasted this, but the comment doesn't serve any important information, which wasn't clear beforehand.